packages/page-experience/README.md

```diff
@@ -0,0 +1,29 @@
+# `page-experience`
+
+** EXPERIMENTAL **
+
+## Usage
+
+```js
+const PageExperienceGuide = require('@ammproject/toolbox-page-experience');
+
+(async ()=> {
+  const pageExperienceGuide = new PageExperienceGuide();
+  const result = pageExperienceGuide.analyze('https://amp.dev');
+  console.log('result')
+})
```

Comment:
is there a reason this is wrapped in `()` but not invoked?

Reply:
not at all...fixed